sdk/keyvault/keyvault-admin/src/generated/models/index.ts

```diff
@@ -8,33 +8,38 @@
 
 import * as coreHttp from "@azure/core-http";
```

Comment:
Regenerating unfortunately added extra noise here in the generated folder due to shifting things around. The meaningful changes are captured in the swagger transforms